kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/SearchChecker.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.good.patterns.challenges.thirdChallenge;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SearchChecker {
+
+    public boolean isFlightStartFromThisAirport(Map<Integer, Flight> mapOfAllFlights, String startAirport) {
```

Comment:
Raczej dałbym pole będące mapą w konstruktorze i metody bez tej may. Metody mogą być statyczne. Powinny zwracać List<Flight> zamiast jakieś boolean, ąłtwiej byś to w dalszej części mógł ogarnąć.